src/golang/lib/collections/artifact_result/types.go

```diff
@@ -6,7 +6,10 @@ import (
 	"github.com/aqueducthq/aqueduct/lib/collections/utils"
 )
 
-type Metadata []map[string]string // Table Schema from Pandas
+type Metadata struct {
+	Schema        []map[string]string // Table Schema from Pandas
+	SystemMetrics map[string]string   // Metrics from the system regarding the op used to create the artifact result. A key/value pair of [metricname]metricvalue e.g. SystemMetric["runtime"] -> "3.65"
```

Comment:
nit: this line is super long, just break it into multiple lines right above this field definition.

Reply:
I thought gofmt would make it style compliant but ill change it